### PR TITLE
Edit and revision of overview & quick start doc

### PR DESCRIPTION
**Pull request by fredlf**

### CODE REVIEW

**Comment by SvenDowideit** [comment]
I _think_ this covers the things i did in #855

File: docs/index.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
----
-layout: default
-title: Compose: Multi-container orchestration for Docker
 page_title: Compose: Multi-container orchestration for Docker
-page_description: Compose: Multi-container orchestration for Docker
-page_keywords: fig, composition, compose, docker
----
+page_description: Introduction and Overview of Compose
+page_keywords: documentation, docs,  docker, compose, orchestration, containers
```
Comment (SvenDowideit) [inline docs/index.md]: can you please put the `---`, layout and title lines back in for now?

**Comment by fredlf** [comment]
Ping @aanand Please review. And is it a problem if we break the fig docs? Which @SvenDowideit just pointed out to me. Head-->desk.

File: docs/index.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
----
+page_description: Introduction and Overview of Compose
+page_keywords: documentation, docs,  docker, compose, orchestration, containers 
 
-# <strong class="strapline">Fast, isolated development environments using Docker.</strong>
 
-Define your app's environment with a `Dockerfile` so it can be reproduced anywhere:
+## Overview
+
+Compose is a tool that allows you to orchestrate multiple Docker containers. With Compose, you can build clusters of containers which provide the resources (services, volumes, etc.) needed to build and run a complete distributed application.
```
Comment (SvenDowideit) [inline docs/index.md]: 80 char wrap :)

File: docs/index.md [hunk 3]
```diff
@@ -73,25 +76,29 @@ if __name__ == "__main__":
...
 
-Next, we want to create a Docker image containing all of our app's dependencies.
-We specify how to build one using a file called `Dockerfile`:
+### Create a Docker image
+
+Now, create a Docker image containing all of your app's dependencies. You
+specify how to build the image using a file called [`Dockerfile`](http://docs.docker.com/reference/builder/):
 
     FROM python:2.7
     ADD . /code
```
Comment (SvenDowideit) [inline docs/index.md]: should we change this to be inline with the best practices?

ie

```
FROM python:2.7
WORKDIR /code
ADD requirements.txt /code/
RUN pip install -r requirements.txt
ADD . /code/
```

File: docs/index.md [hunk 4]
```diff
@@ -107,14 +114,18 @@ We then define a set of services using `docker-compose.yml`:
...
 This defines two services:
 
- - `web`, which is built from `Dockerfile` in the current directory. It also says
-   to run the command `python app.py` inside the image, forward the exposed port
-   5000 on the container to port 5000 on the host machine, connect up the Redis
-   service, and mount the current directory inside the container so we can work
-   on code without having to rebuild the image.
- - `redis`, which uses the public image [redis](https://registry.hub.docker.com/_/redis/). 
+ - `web`, which is built from the `Dockerfile` in the current directory. It also
+ says to run the command `python app.py` inside the image, forward the exposed
```
Comment (SvenDowideit) [inline docs/index.md]: I think these need to be indented as before, otherwise they won't be a part of the bullet point.

File: docs/index.md [hunk 5]
```diff
@@ -124,11 +135,12 @@ Now if we run `docker-compose up`, it'll pull a Redis image, build an image for
     redis_1 | [8] 02 Jan 18:43:35.576 # Server started, Redis version 2.8.3
     web_1   |  * Running on http://0.0.0.0:5000/
 
-The web app should now be listening on port 5000 on your docker daemon (if you're
-using boot2docker, `boot2docker ip` will tell you its address).
+The web app should now be listening on port 5000 on your docker daemon (if
```
Comment (SvenDowideit) [inline docs/index.md]: on your Docker daemon host

caps, and the daemon doesn't have a port to listen on, its the host

**Comment by SvenDowideit** [comment]
ok, I might make a follow on PR based on this to fix up the things I've found so @fredlf can continue work on it tomorrow :)

@aanand let me know what you'd like us to do, and it'll be done

**Comment by SvenDowideit** [comment]
replaced by #902

**Pull request closed without merging**